web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -1321,6 +1321,18 @@ class LiftRules() extends Factory with FormVendor with LazyLoggable {
   val snippets = RulesSeq[SnippetPF]
 
   /**
+   * Handles the parsing of template content into NodeSeqs.
+   */
+  @volatile var contentParsers:Seq[ContentParser] = Seq(
+    new ContentParser {
```

Comment:
Yes... in fact I think that was the intention, but I forgot to go back and do it.